auth/TESLA/Sender_Notary.m

```diff
@@ -0,0 +1,111 @@
+classdef Sender_Notary < handle
+    % SENDER_NOTARY Summary of this class goes here
+    %   Detailed explanation goes here
+
+    properties (Access = public)
+        key_chain_length = 100
+        key_disclosure_delay = 5
+        key_chain_function = @HashingWrappers.sha_256
```

Comment:
Note, the keychain function is shaped-256, _truncated to 128-bits_. You should create a static method in this class that calls `HashingWrappers.sha_256`, but also truncates to 128-bits. Then add the local static function handle here

Reply:
I'm a little confused by this. so the truncate method should be a static method in the HashingWrappers class that gets called here?